### ISSUE TITLE
Unable to mark, post, or PM, with "signature is expired" error

### ISSUE THREAD

**Issue by matt-s0101745** [knighty/fimfiction-issues#332]
I read a few chapters of a story and they didn't stay read. I tried to manually mark them as read, but I got an error message: "Signature is expired, the time on your computer or device is likely incorrect. The difference is 2533 weeks". I resynchronised my computer's time and it still occurs. It also occurs when I try to post a comment. It also occurs when I try to send a PM, but not when I try to add a story to my library. How do I fix this?
I run Firefox 18.0.2 on Windows 7
(You may need to contact me off-GitHub. I am "matt s0101745" on fimfiction or matt.s0101745@yahoo.com)

**Comment by sweetaibelle** [comment]
I'm thinking log out of your fimfiction account and back in, and maybe clear cache and cookies if that doesn't help...

**Comment by matt-s0101745** [comment]
I found a workaround by using a new alternate browser (that isn't as much of a memory hog and therefore last resort solution as IE), installed some add-ons, and found a similar error occurring there too (Opera 54.0.something on Windows 7), and resolved it by clearing the cookies.
I tried clearing the cookies in Firefox as well, but it didn't work. I guess I'll try clearing the cache, but it isn't as urgent now.

**Comment by matt-s0101745** [comment]
Now I've tried all three, and none of them worked.

**Comment by SomewhatDamaged** [comment]
This can be caused by a few things. The first and foremost of them being a plugin interfering with your HTTPS timing (usually an overzealous antivirus).

I'd suggest closing all browsers/apps except chrome, running in an incognito window, and disable your AV completely. Then try accessing the site.

If this works, it's a sign that something that got turned off (AV/addons) is messing with the site. If it doesn't, it may be something else (are you using a proxy? Is your ISP running a transparent proxy?).

Narrow down what it is by turning on plugins one at a time (start with your AV) to see which one is the culprit.

**Comment by Roshiyu676** [comment]
Firefox updated, and now I'm suffering the same problem. You ever find a fix for Firefox?

**Comment by matt-s0101745** [comment]
No. I had assumed that it was because I am using version 18.0.something instead of an up-to-date version.
I just decided to work by my own memory (aided by a session manager and the fact that I use the 'download story' option*) to remember which stories and chapters I read. Also, I do not write stories or post many comments, so I can stand using a secondary browser (in my case, Opera) to do those things.

(*In fact, I was the one who asked for it to be put back at the bottom of the story box where it was before.)

However, if this is the same problem, there is a symptom that I only noticed later: Firefox's history fills up with bookmark cookies (eg, "bookmark1280800") that it will not delete on its own; it sends them with every page request until the page requests become too big to send and you have to delete them manually to browse any more pages on the site.
(If it is still the same layout of Firefox, it is in the 'options' window, under the 'privacy' tab, under 'history', 'remove individual cookies'. With my setup, all of the bookmark* cookies are contiguous, but you might need to order them by cookie name to select them all at once.)
This only happens every few weeks for me (no doubt why I only noticed it later), so it is even less onerous than not being able to post comments in Firefox, but it is a symptom.

**Comment by Roshiyu676** [comment]
Just tried that, and boy oh boy was that a mess to clean up. It cleared my history, cookies, and of course my 80+ tabs. Which I found out only after closing down firefox and restarting it. :/

In the end I had to go all the way to my old back-ups, from when I did a clean Win10 install a year ago and copy over that firefox profile. Saved 50ish of my tabs from that. Add in a couple more from memory...

On the plus side FIMfiction and all the other websites that were glitching out are now working again. It only cost me 20ish tabs. @.@
